Antidote/ActiveCallViewController.m

```diff
@@ -240,11 +242,35 @@ - (void)installConstraints
     }];
 
     [self.videoContainerView makeConstraints:^(MASConstraintMaker *make) {
-        make.top.equalTo(self.topViewContainer.bottom);
-        make.left.equalTo(self.view);
-        make.right.equalTo(self.view);
-        make.bottom.equalTo(self.compactControlsView.top);
+        make.edges.equalTo(self.view);
     }];
+
+    [self updateUIForInterfaceOrientation:self.interfaceOrientation];
+}
+
+- (void)willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration
+{
+    [self updateUIForInterfaceOrientation:toInterfaceOrientation];
+
+    [super willRotateToInterfaceOrientation:toInterfaceOrientation duration:duration];
+}
+
+- (void)updateUIForInterfaceOrientation:(UIInterfaceOrientation)interfaceOrientation
+{
+    const BOOL isPortrait = UIInterfaceOrientationIsPortrait(interfaceOrientation);
+
+    const CGFloat compactControlsXIndents = isPortrait ? 0.0 : kCompactLandScapeXIndent;
+
+    [self.compactControlsView updateConstraints:^(MASConstraintMaker *make) {
+        make.left.equalTo(self.view).with.offset(compactControlsXIndents);
+        make.right.equalTo(self.view).with.offset(-compactControlsXIndents);
+    }];
+
+    const BOOL isCompacted = ! isPortrait ||  [self videoViewIsShown];
+    const BOOL isExpanded = ! isCompacted;
```

Comment:
In this case there is no need in getting `isExpanded`, you can use `isCompacted` to set views to hidden.